Sources/ObjectForm/row/ButtonRow.swift

```diff
@@ -9,7 +9,7 @@ import Foundation
 import UIKit
 
 /// Model for a row that is a button
-class ButtonRow: BaseRow {
```

Comment:
Unfortunately this PR doesn't build, let me make another PR to fix this. Thanks for the contribution.

Reply:
Oh wow, I replace `class` by `open` lol 😅

Let me fix that real quick